### PR TITLE
Updating Patrol Route natives

### PR DESCRIPTION
**Pull request by XenoS-ITA**
i documented updating information on the two native **ADD_PATROL_ROUTE_NODE** and **ADD_PATROL_ROUTE_LINK**.
i attach below the code i used to test the information i wrote:

```lua
local _0 = vector3(-1084.17, -477.15, 36.63)
local _1 = vector3(-1083.59, -485.5, 36.58)
local _2 = vector3(-1077.57, -478.0, 36.67)
local _3 = vector3(-1082.03, -463.55, 36.58)

RequestModel(`mp_m_securoguard_01`)

while not HasModelLoaded(`mp_m_securoguard_01`) do
    Citizen.Wait(1)
end

local ped= CreatePed(`mp_m_securoguard_01`, _0, 0.0, true)

-- Create the patrol
OpenPatrolRoute("miss_test")
AddPatrolRouteNode(0, "StandGuard", _0, _1, 5000)
AddPatrolRouteNode(1, "StandGuard", _1, _2, 5000)
AddPatrolRouteNode(2, "StandGuard", _2, _3, 5000)
AddPatrolRouteNode(3, "StandGuard", _3, _0, 5000)
AddPatrolRouteLink(0, 1)
AddPatrolRouteLink(1, 2)
AddPatrolRouteLink(2, 3)
AddPatrolRouteLink(3, 0)
ClosePatrolRoute()
CreatePatrolRoute()

-- Draw Lines to see the route
Citizen.CreateThread(function()
    while true do
        DrawLine(_0, _1, 255, 0, 0, 255)
        DrawLine(_1, _2, 0, 255, 0, 255)
        DrawLine(_2, _3, 0, 0, 255, 255)
        DrawLine(_3, _0, 0, 255, 255, 255)
        Citizen.Wait(1)
    end
end)

TaskPatrol(ped, "miss_test", 1, 0, 1)
```

### CODE REVIEW

File: TASK/AddPatrolRouteNode.md [hunk 1]
```diff
@@ -5,30 +5,25 @@ ns: TASK
...
-* **p8**: 
+* **id**: is an integer that "identifies" the route node for linking to the route with ADD_PATROL_ROUTE_LINK.
+* **guardScenario**: can be "WORLD_HUMAN_GUARD_STAND" or "StandGuard"
+* **x1**: destination
+* **y1**: destination
+* **z1**: destination
+* **x2**: coordinates to which the ped should look
+* **y2**: coordinates to which the ped should look
+* **z2**: coordinates to which the ped should look
+* **waitTime**:
```
Comment (4mmonium) [inline TASK/AddPatrolRouteNode.md:22]: Just nitpicking here, but maybe clarify that the waitTime should be in milliseconds :-)

Comment (XenoS-ITA) [reply TASK/AddPatrolRouteNode.md]: you are right, i have updated the description

File: TASK/AddPatrolRouteLink.md [hunk 2]
```diff
@@ -5,11 +5,26 @@ ns: TASK
...
+connects 2 route nodes to make it clear how the ped should move
 
 ## Parameters
-* **p0**: 
-* **p1**: 
+* **id1**: 
+* **id2**: 
 
+## Examples
+![image](https://user-images.githubusercontent.com/55803068/188470866-c32c6a9f-a25d-4772-9b18-5be46e2c14a1.png)
```
Comment (4mmonium) [inline TASK/AddPatrolRouteLink.md:18]: Hey, could you please put the image in the description under 'connects 2 route nodes...' and point out that the image represents this example?

File: TASK/AddPatrolRouteLink.md [hunk 3]
```diff
@@ -5,11 +5,26 @@ ns: TASK
...
+
+``` lua
```
Comment (4mmonium) [inline TASK/AddPatrolRouteLink.md:20]: Maybe I'm just nitpicking here, but could you please remove the space between the backticks and lua?

Replace with:
```
```lua
```

Thanks! 🙂

**Review by 4mmonium — APPROVED** [review]
Looks good to me, thank you!

**Pull request merged**